lightningd/pay.c

```diff
@@ -1116,9 +1117,22 @@ send_payment_core(struct lightningd *ld,
 		return command_failed(cmd, data);
 	}
 
+
+	/* BOLT 2 (0539ad868a263040087d2790684f69fb28d3fa97):
```

Comment:
use bolt-xxx #2, which is how we do this.  In this case, use -jamming I'd say, instead of a particular commit?

Reply:
Ok yeah moving this to `BIP-jamming #4`, but make sense